npm/vue2/src/index.ts

```diff
@@ -410,7 +412,7 @@ export const mount = (
 
     Cypress.vue = VTUWrapper.vm
     Cypress.vueWrapper = VTUWrapper
-  })
+  }).wait(0, { log: false })
```

Comment:
Do we need the wait here?

Reply:
No we do not. An oversight on my end from the previous implementation. This has been removed in the latest commit.